bundles/org.openhab.binding.haywardomnilogic/README.md

```diff
@@ -0,0 +1,401 @@
+Number   PoolHeaterEnable "Power [%s]"                                          { channel = "haywardomnilogic:heater:2ee76053:33:heaterEnable"}
+Number   PoolHeaterPriority "Priority [%s]"                                     { channel = "haywardomnilogic:heater:2ee76053:33:heaterPriority"}
+Number   PoolHeaterMaintainFor "Maintain For [%s]"                              { channel = "haywardomnilogic:heater:2ee76053:33:heaterMaintainFor"}
+
+Switch   PoolLightEnable "Enable"                                               { channel = "haywardomnilogic:colorlogic:2ee76053:38:colorLogicLightEnable"}
+Number   PoolLightState "State [JS(haywardLightState.js):%s]"                   { channel = "haywardomnilogic:colorlogic:2ee76053:38:colorLogicLightState"}
+Number   PoolLightCurrentShow "Show"                                            { channel = "haywardomnilogic:colorlogic:2ee76053:38:colorLogicLightCurrentShow"}
+Number   PoolLightSpeed "Speed [%s]"                                            { channel = "haywardomnilogic:colorlogic:2ee76053:38:colorLogicLightSpeed"}
+Number   PoolLightBrightness "Brightness [%s]"                                  { channel = "haywardomnilogic:colorlogic:2ee76053:38:colorLogicLightBrightness"}
+
+Switch   PoolFilterEnable "Power"                                               { channel = "haywardomnilogic:filter:2ee76053:31:filterPowerSwitch"}
+Number   PoolFilterValve "Valve Position [%s]"                                  { channel = "haywardomnilogic:filter:2ee76053:31:filterValvePosition"}
+Number   PoolFilterSpeed "Speed[%1.0f %%]"                                      { channel = "haywardomnilogic:filter:2ee76053:31:filterSpeed"}
+Number   PoolFilterState "State [JS(haywardFilterState.js):%s]"                 { channel = "haywardomnilogic:filter:2ee76053:31:filterState"}
+Number   PoolFilterCall "Why Running [JS(haywardFilterWhyRunning.js):%s]"       { channel = "haywardomnilogic:filter:2ee76053:31:filterWhyFilterIsOn"}
+Number   PoolFilterFP "Freeze Protection"                                       { channel = "haywardomnilogic:filter:2ee76053:31:filterFpOverride"}
+
+Switch   PoolVirtualHeaterEnable "Power"                                        { channel = "haywardomnilogic:virtualHeater:2ee76053:32:virtualHeaterEnable"}
+Number   PoolVirtualHeaterTemp "Temp [%1.0f °F]"                                { channel = "haywardomnilogic:virtualHeater:2ee76053:32:virtualHeaterCurrentSetpoint"}
+  
+Switch   PoolRelay1 "Deck Jets"                                                 { channel = "haywardomnilogic:relay:2ee76053:37:relayState"}
+Switch   PoolRelay2 "Vacuum"                                                    { channel = "haywardomnilogic:relay:2ee76053:36:relayState"}
+```
+
+### transforms:
+
+These transform will decode the integers provided from Hayward into text descriptions.  Please note this is a work in progress.
+If you have any descriptions to add, please email to the code owner for updates to this page.
```

Comment:
Are these mappings user configurable? Otherwise these should be in the xml as options as fwolter mentions. See the documentation about this: https://www.openhab.org/docs/developer/bindings/thing-xml.html#state-description Note that these channels must be of item-type String as mentioned in the documentation.